vue/dynamicforms/src/index.ts

```diff
@@ -54,6 +55,7 @@ export function createDynamicForms(options: DynamicFormsOptions = defaultOptions
 
       // import all global instances that we need for vuetify to work
       Object.entries(VuetifyComponents).map(([name, component]) => app.component(name, component));
+      app.component('df-app', DfApp);
```

Comment:
This probably raises an error?

You can change 'df-app' to 'DfApp'. This should fix the error and preserve functionality.